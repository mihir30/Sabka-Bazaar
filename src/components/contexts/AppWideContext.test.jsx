import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import AppWideContextProvider from './AppWideContext';

describe("App wide context provider component", () => {
    it('renders the component properly', () => {
        render(<AppWideContextProvider/>);
    });

    it('should have the right app wide context provider component snapshot', () => {
        const tree = renderer.create(<AppWideContextProvider/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

});