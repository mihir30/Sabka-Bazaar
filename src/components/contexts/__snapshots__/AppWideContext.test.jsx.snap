// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App wide context provider component should have the right app wide context provider component snapshot 1`] = `null`;
